d3rlpy/logging/wandb_adapter.py

```diff
@@ -0,0 +1,73 @@
+import wandb
```

Comment:
To allow users to use d3rlpy without installing wandb, can you import the package just like here?
https://github.com/takuseno/d3rlpy/blob/06dd7a7698bf68af0645afeb62f148aad921a9bd/d3rlpy/logging/tensorboard_adapter.py#L30